src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -70,4 +71,39 @@ export class IeObjectsController {
 			userHasAccessToSpace
 		);
 	}
+
+	@Post()
+	@RequireAllPermissions(Permission.SEARCH_OBJECTS)
```

Comment:
anonymous users will also be allowed to search, so we cannot have a permission check here
```suggestion
```